h2/src/main/org/h2/util/ToDate.java

```diff
@@ -0,0 +1,63 @@
+package org.h2.util;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+import org.h2.util.ToDateTokenizer.FormatTokenEnum;
+
+/**
+ * Emulates Oracle's TO_DATE function.<br>
+ * Main class
+ */
+public class ToDate {
+    public enum ToDateFunctionName {
+        TO_DATE("DD MON YYYY"), TO_TIMESTAMP("DD MON YYYY HH:MI:SS");
+
+        private final String defaultFormatStr;
+
+        ToDateFunctionName(final String defaultFormatStr) {
+            this.defaultFormatStr = defaultFormatStr;
+        }
+
+        String getDefaultFormatStr() {
+            return defaultFormatStr;
+        }
+    };
```

Comment:
why is enum class necessary? Why not just pass the string to the ToDateParams class?